src/Symfony/Component/Routing/Tests/Matcher/UrlMatcherTest.php

```diff
@@ -396,7 +396,7 @@ public function testMissingTrailingSlash()
 
     public function testExtraTrailingSlash()
     {
-        $this->getExpectedException() ?: $this->expectException(ResourceNotFoundException::class);
+        $this->expectException(ResourceNotFoundException::class);
```

Comment:
while on it, can you please move them just before the throwing code-part?

Reply:
We can do that in a follow-up.